src/client/public/locale/de/common.json

```diff
@@ -86,6 +86,7 @@
   "enabling": "{{what}} aktivieren",
   "english": "Englisch",
   "errorAttention": "Achtung",
+  "errorDuplicatedUploadPerBorehole": "Diese Datei wurde bereits für diese Bohrung hochgeladen und kann nicht ein zweites Mal hochgeladen werden.",
```

Comment:
REQ: Dieser neue Eintrag muss noch in das `common.json`-File, [welches ](https://github.com/geoadmin/suite-bdms/blob/main/src/client/public/locale/common.json)direkt unter `/locale` liegt, hinzugefügt werden.

Reply:
@Lehats Dieser Punkt muss noch erledigt werden.

Reply:
@danjov Meintest du nicht den Eintrag im `common.json` von diesem Committ?
https://github.com/geoadmin/suite-bdms/pull/510/commits/55798c504af91054544da946fdafdb015d7be534

Reply:
@Lehats Stimmt, habe das übersehen 👍🏻.